### PR TITLE
move readBom methods to Bom class

### PR DESCRIPTION
**Pull request by elharo**
@suztomo

### CODE REVIEW

File: dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/BomTest.java [hunk 1]
```diff
@@ -26,19 +26,19 @@
...
+import com.google.cloud.tools.opensource.dependencies.Bom;
 import com.google.cloud.tools.opensource.dependencies.MavenRepositoryException;
-import com.google.cloud.tools.opensource.dependencies.RepositoryUtility;
 
 public class BomTest {
 
   private static final Path CLOUD_OSS_BOM_PATH =
-      Paths.get("..", "boms", "cloud-oss-bom", "pom.xml").toAbsolutePath();
-
+      Paths.get("..", "boms", "cloud-oss-bom", "pom.xml").toAbsolutePath();
```
Comment (suztomo) [inline dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/BomTest.java:35]: Unnecessary space at the end of line/

File: dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/Bom.java [hunk 2]
```diff
@@ -59,6 +72,75 @@ public String getCoordinates() {
...
+        .map(Dependency::getArtifact)
+        .filter(artifact -> !shouldSkipBomMember(artifact))
+        .collect(ImmutableList.toImmutableList());
+    
+    Bom bom = new Bom(coordinates, artifacts);
+    return bom;
+  }
+
+  /**
+   * Parse the dependencyManagement section of an artifact and return the artifacts included there.
```
Comment (suztomo) [inline dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/Bom.java:97]:
```suggestion
   * Parses the dependencyManagement section of an artifact and returns the artifacts included there.
```

File: dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassPathBuilderTest.java [hunk 3]
```diff
@@ -19,8 +19,7 @@
 import static com.google.cloud.tools.opensource.classpath.TestHelper.COORDINATES;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
-
-import com.google.cloud.tools.opensource.dependencies.RepositoryUtility;
```
Comment (suztomo) [inline dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassPathBuilderTest.java:23]: Insert an empty line between static imports and non-static imports.

File: dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/Bom.java (hunk 2 above)
Comment (elharo) [reply dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/Bom.java:97]: done

File: dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassPathBuilderTest.java (hunk 3 above)
Comment (elharo) [reply dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassPathBuilderTest.java:23]: done

**Pull request merged**